app/src/main/java/org/cru/godtools/dagger/ServicesModule.kt

```diff
@@ -42,7 +40,7 @@ abstract class ServicesModule {
     companion object {
         @Provides
         @Singleton
-        fun workManager(@ApplicationContext context: Context, workerFactory: DaggerWorkerFactory): WorkManager {
+        fun workManager(@ApplicationContext context: Context, workerFactory: HiltWorkerFactory): WorkManager {
```

Comment:
What's different between the ```DaggerWorkerFactory``` and the ```HiltWorkerFactory```? Or does it just let you use Hilt instead of Dagger?

Reply:
`DaggerWorkerFactory` is a `WorkerFactory` I implemented to allow Dagger to manage Workers.
`HiltWorkerFactory` is a `WorkerFactory` from AndroidX Hilt which we won't have to maintain.